docs for httpx v0.1.0 (#1120)
grafana/k6-docs: src/data/markdown/docs/20 jslib/01 jslib/02 httpx/15 options(url, [body], [params]).md

`session.options(url, body, params)` makes an OPTIONS request. Only the first parameter is required

| Parameter         | Type                                                                                      | Description                                                                            |
|-------------------|-------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------|
| url               | string                                                                                    | HTTP URL. If baseURL is set, provide only path.                                        |

> | body (optional)   | null / string / object / ArrayBuffer / [SharedArray](/javascript-api/k6-data/sharedarray) | Request body; objects will be `x-www-form-urlencoded`. Set to `null` to omit the body. |

Reviewer: ```suggestion
| body (optional)   | null / string / object / ArrayBuffer / [SharedArray](/javascript-api/k6-data/sharedarray) | Request body; objects are `x-www-form-urlencoded`. To omit the body, set to `null`. |
```
